include/lsst/afw/geom/EdgeRolloffFunctor.h

```diff
@@ -0,0 +1,67 @@
+// -*- lsst-c++ -*-
+
+/* 
+ * LSST Data Management System
+ * Copyright 2015 LSST Corporation.
+ * 
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the LSST License Statement and 
+ * the GNU General Public License along with this program.  If not, 
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+/**
+ * \file 
+ * @brief Functor to describe edge rolloff effect in CCDs.
+ */
+
+#ifndef LSST_AFW_GEOM_EDGEROLLOFFFUNCTOR_H
+#define LSST_AFW_GEOM_EDGEROLLOFFFUNCTOR_H
+
+#include "lsst/afw/geom/Functor.h"
+
+namespace lsst {
+namespace afw {
+namespace geom {
+
+/// @brief Stubbs parameterization of edge rolloff effect in LSST CCDs.
```

Comment:
Specifying a valid range for x is problematic.  It's true that for operator()(x), it wouldn't make sense to provide an x outside of [0, xmax], but when computing the inverse via Newton-Raphson, the initial guess is the provided y value, which often lies outside of [0, xmax], and the range of that value depends on amplitude, xmax, and xscale. Since inverse calls operator()(...) as well as derivative(...), it can potentially violate that range constraint.   One could restrict the initial estimate to lie within the [0, xmax] range, but then that would involve re-implementing inverse(...) and also wouldn't guarantee that the subsequent trial values would stay in range.  Unless there is a divide by zero issue or some other floating point problem, I don't think it is desirable to enforce a validity range on x.